CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## [5.1.7] - 2022-12-13
+
+### Changed
+
+Add timeout option
```

Comment:
Can you be a bit more descriptive of where this was added?

Also, can you please update the examples in `README.md` to include this?

Reply:
Added the param in the readme example